style-guide/_colors.scss

```diff
@@ -77,16 +77,9 @@ $color_grey_quote:           $palette_grey_text;
 $color_grey_text:            $palette_grey_text;
 
 // My Yoast colors.
-$color_chevron:              $palette_grey_medium_dark;
-$color_round_button:         $palette_green_medium_light;
```

Comment:
wow, based on the names of some of these colors, I was convinced they were used in certain spots that were unedited. But It seems you were right. Looking through the master branch, they indeed aren't used in those spots. Makes me wonder why some of those were made in the first place.